### PR TITLE
Update titles

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/docs/webhooks.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 # Webhooks
 
-A webhook directs information from one application to another in real-time and triggers the receiving application to execute a specific process in response. Choreo allows you to create webhooks for platforms such as Github, Slack, and Google Calendar using the integrated pro-code or low-code editor. For example, you can design a webhook to notify a Github user via mail each time an issue is assigned to him/her.
+A webhook directs information from one application to another in real-time and triggers the receiving application to execute a specific process in response. Choreo allows you to create webhooks for platforms such as Github, Slack, and Google Calendar using the integrated pro-code or low-code editor.
```
Comment (Nashaath) [inline en/docs/webhooks.md:3]: Should we capitalize the W in the term Webhook as it is a name of a component in Choreo?

File: en/docs/webhooks.md [hunk 2]
```diff
@@ -20,22 +26,14 @@ You can implement the logic and test it locally in the code server. After that y
 
 You can also use the Choreo built-in connectors in the webhook configuration.
```
Comment (Nashaath) [inline en/docs/webhooks.md:27]: Is it correct to say "built-in connectors"? What we want to say is that it is possible to use any of the connectors that Choreo supports so far right?

Comment (RukshiW) [reply en/docs/webhooks.md]: Changed this sentence.

File: en/docs/webhooks.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/webhooks.md:3]: Used capital W when it is referred to as the "Webhook component".

**Review by Nashaath — APPROVED** [review]

**Pull request merged**